content/blog/how-to-integrate-with-mainnet-protocols/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "How to Integrate with Ethereum Mainnet DeFi Protocols"
+date: "2020-04-24"
+description: "Wanna try build a dapp that interacts with multiple mainnet protocols but don't know where to start? Read on!"
+---
+
+While building out [Dedge](https://dedge.exchange), one of the first problems we encountered was:
+
+> How and where do we start integrating with mainnet DeFi protocols?
+
+Ideally, we would like to fork off the current mainnet state into a sandboxed environment, and develop/test our smart contracts from there. The emphasis on forking off the mainnet state it provides a couple key features:
+
+1. Liquidity automatically configured on other DeFi procotols e.g. Aave/Compound/Compound.
+2. You're almost guaranteed every major procotol will be deployed on mainnet.
+    - If you're interacting with multiple protocols, they might not all be available on the same testnet e.g. Rinkeby
+3. As you're forking off the mainnet state, all mainnet contract addresses will be the same in your private testnet.
+4. Increases your development velocity as you can just focus on developing your application, not writing scripts to deploy the protocols to your private testnet.
```

Comment:
```suggestion
4. Your development velocity is significantly increased because you don't have to worry about deploying existing protocols onto your test network.
```